templates/documents/repositories.html

```diff
@@ -0,0 +1,47 @@
+{# Generic template to render a list of objects with a 'new' button #}
+{% extends 'web/app/app_base.html' %}
+{% block app %}
+    {% load i18n %}
+    <div class="app-card" x-data>
+        <div class="grid grid-cols-6">
+            <div class="col-span-5">
```

Comment:
a better option here might be to use `flex` with `justify-between` which deals better with different screen sizes